src/test/java/org/folio/rest/impl/UsersAPITest.java

```diff
@@ -174,26 +164,41 @@ void checkAddressTypeValidCanHandleException(VertxTestContext context) {
   }
 
   @Test
-  void checkAddressTypeValidCanHandleNullPostgresClient(Vertx vertx, VertxTestContext context) {
-    Future<Boolean> future = new UsersAPI().checkAddressTypeValid("myId", vertx.getOrCreateContext(),
-        /* postgresClient */ null);
+  void checkAddressTypeValidCanHandleNullPostgresClient(VertxTestContext context) {
+    final var future = new UsersAPI()
+      .checkAddressTypeValid("myId", null);
+
     future.onComplete(context.failing(e -> context.verify(() -> {
       assertThat(future.cause(), is(instanceOf(NullPointerException.class)));
       context.completeNow();
     })));
   }
 
   @Test
-  void checkAllAddressTypesValidCanHandleNullPostgresClient(Vertx vertx, VertxTestContext context) {
+  void checkAllAddressTypesValidCanHandleNullPostgresClient(VertxTestContext context) {
     Address address = new Address().withAddressTypeId("someAddressTypeId");
     Personal personal = new Personal().withAddresses(Collections.singletonList(address));
     User user = new User().withPersonal(personal);
-    Future<Boolean> future = new UsersAPI().checkAllAddressTypesValid(user, vertx.getOrCreateContext(),
-        /* postgresClient */ null);
+    Future<Boolean> future = new UsersAPI().checkAllAddressTypesValid(user, null);
     future.onComplete(context.failing(e -> context.verify(() -> {
       assertThat(future.cause(), is(instanceOf(NullPointerException.class)));
       context.completeNow();
     })));
   }
+
+  private void checkAddressTypeValid(VertxTestContext context,
+    Future<Results<Object>> result, Handler<Throwable> handler) {
+
+    final var postgresClient = mock(PostgresClient.class);
+
+    when(postgresClient.get(anyString(), any(), any(Criterion.class), anyBoolean()))
+      .thenReturn(result);
+
+    final var future = new UsersAPI()
+      .checkAddressTypeValid("someAddressTypeId", postgresClient);
+
+    future.onComplete(context.failing(e ->
+      context.verify(() -> handler.handle(future.cause()))));
```

Comment:
```
    future.onComplete(context.failing(e -> handler.handle(e)));
```
or
```
    future.onComplete(context.failing(handler::handle));
```
context.failing has context.verify included, therefore context.verify is not needed.
context.failing provides the value of future.cause() as e.